scripts/execute-test

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+TEST=$1
+
+if [ "" == "${TEST}" ]; then
+	echo "You need to specify at least one test"
+	exit 1
+fi
+
+NETWORK=$(docker network ls --filter name=enveloping-tests-net -q)
+if [ "${NETWORK}" == "" ]; then
```

Comment:
Please don't mix `bash`-specific syntax with portable shell syntax. Either use `[ "${NETWORK}" = "" ]` (single `=`) or `[[ "${NETWORK}" == "" ]]` (double `[[` and `]]`). I prefer using `[ -z "$NETWORK" ]`, which is the idiomatic way to ask if a string is empty in `sh`.